addons/mass_mailing/static/src/xml/mass_mailing_mobile_preview.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <t t-name="mass_mailing.MobilePreviewDialog">
+        <Dialog size="'lg'" title="title">
```

Comment:
title not useful here since you set the slot header

Reply:
also "lg" is the default size I think

Reply:
wow ... of course. It's ok for "lg" size by default.